src/frontend/Sider/Behandling/Brev/brevUtils.ts

```diff
@@ -8,3 +10,25 @@ export const idEllerFritekst = (valg?: Valg): string | undefined => {
             return 'fritekst';
     }
 };
+
+export const finnSanityMappe = (
+    behandlingstype: BehandlingType,
+    vedtakType: TypeVedtak
+): string => {
+    if (behandlingstype === BehandlingType.REVURDERING) {
+        return 'REVURDERING';
+    }
+    return typeVedtakTilSanitytype(vedtakType);
+};
+
+/**
+ * Mapper fra type vedtak til typen malen er definert som i Sanity.
+ */
+export const typeVedtakTilSanitytype = (type: TypeVedtak): string => {
```

Comment:
Tror export kan fjernes her nå?
```suggestion
const typeVedtakTilSanitytype = (type: TypeVedtak): string => {
```